controller-client/src/main/java/org/jboss/as/controller/client/OperationBuilder.java

```diff
@@ -146,25 +169,25 @@ public static OperationBuilder create(final ModelNode operation, final boolean a
     // Wrap the FIS in a streamEntry so that the controller-client has access to the underlying File
     private static class FileStreamEntry extends FilterInputStream implements InputStreamEntry {
 
-        private final File file;
+        private final Path file;
         private FileStreamEntry(final File file) throws IOException {
-            super(new FileInputStream(file)); // This stream will get closed regardless of autoClose
+            this(file.toPath());
+        }
+
+        private FileStreamEntry(final Path file) throws IOException {
+            super(Files.newInputStream(file)); // This stream will get closed regardless of autoClose
             this.file = file;
         }
 
         @Override
         public int initialize() throws IOException {
-            return (int) file.length();
+            return (int) Files.size(file);
         }
 
         @Override
         public void copyStream(final DataOutput output) throws IOException {
-            final FileInputStream is = new FileInputStream(file);
-            try {
-                StreamUtils.copyStream(is, output);
-                is.close();
-            } finally {
-                StreamUtils.safeClose(is);
+            try (InputStream in = new BufferedInputStream(Files.newInputStream(file))) {
```

Comment:
Actually this one is ok, since L165 was closing the IS and that can throw an exception.